userspace/libsinsp/logger.cpp

```diff
@@ -205,3 +271,84 @@ char* sinsp_logger::format(const char* fmt, ...)
 
 	return m_tbuf;
 }
+
+const char *sinsp_logger::encode_severity(const severity sev)
+{
```

Comment:
If there are ever threads that continue running after the `main()` returns, then these static objects might end up getting destroyed while those threads are still making calls to the logging API.  If we know for sure that no thread will still be running when `main()` returns, then what you have is fine.  Otherwise, you might consider dynamically allocating the strings so that they don't get destroyed at static deinitialization time.